tests/cxx/typedef_in_template.cc

```diff
@@ -122,6 +122,12 @@ struct Identity {
   struct Inner {
     using Type = T;
   };
+
+  static T t;
+  using SugaredType = decltype(t);
+
+  template <int>
+  using AliasTemplate = T;
```

Comment:
I'm not sure that the test named "Typedef in template" is the best place for alias template testing, but very convenient.

Reply:
I think they're sufficiently related that it makes sense.